### ISSUE TITLE
Any plans to integrate parenscript or similar?

### ISSUE THREAD

**Issue by bqv** [akirakyle/emacs-webkit#5]
Hypothetically

**Comment by akirakyle** [comment]
Not really. I personally think trying to work with parenscript for only running javascript just makes things needlessly complicated since you still need to know the whole DOM api to do anything interesting. Perhaps there's a compelling enough argument to make a parenscript equivalent in elisp, but for now I'm sticking with using JSON to communicated between JS and elisp.

**Comment by bqv** [comment]
I feel it'd be cool to have something more lisp-native. Understandable, though
